src/config.c

```diff
@@ -359,6 +360,8 @@ static int config_callback(void* user, const char* section, const char* name, co
         config->pid_file = strdup(value);
     } else if (NAME_MATCH("input_counter")) {
         config->input_counter = strdup(value);
+    } else if (NAME_MATCH("metrics_prefix")) {
```

Comment:
I wonder if it's a good idea to avoid confusion with "normal" metrics and call this `statsite_internal_metrics_prefix` ?

Reply:
yeah, i kind of sat in the middle because i didn't want the super long name, i might change to internal_metrics_prefix